browser/src/control/Control.PartsPreview.js

```diff
@@ -633,6 +633,7 @@ L.Control.PartsPreview = L.Control.extend({
 			if (this._previewTiles[e.id]) {
 				this._previewTiles[e.id].src = e.tile.src;
 				this._previewTiles[e.id].fetched = true;
+				console.log('PREVIEW: part fetched : ' + e.id);
```

Comment:
There is an app specific wrapper: window.app.console.log app. that can be disabled IIRC - we should use that consistently everywhere surely ?

Reply:
Sure, I will change it